0016-intern-like-system.md

```diff
@@ -0,0 +1,53 @@
+# Intern Like system
+
+* Status: proposed 
+<!-- {proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)} --> <!-- optional -->
+* Input: Welcome from anyone!
+* Deciders: {list everyone involved in the decision using github handle}
+<!-- Find deciders here: https://github.com/orgs/Greenstand/people  -->
+* Date: 2022-09-09 <!-- optional -->
+
+
+## Context and Problem Statement
+
+The Facebook Like button on the web map currently has some challenges with the Eurpean Region, https://developers.facebook.com/docs/plugins/like-button
+
+Another issue is that we are not able to create a custom Like button in the Greenstand style, https://github.com/Greenstand/treetracker-web-map-client/pull/395
+
+## Decision Drivers <!-- optional -->
+
+* {driver 1, e.g., a force, facing concern, …}
+
+## Considered Options
+
+* Option #1 - Intern Like system, to save likes on trees/entities directly
+* Option #2 - Your suggestions are welcome
```

Comment:
Option #2 – Internal Like system linked with Keycloak user authentication system

Reply:
The need to log in before liking/saving an entity is a very common pattern, so would be expected by most users. The "like" button is a great way to ease users into creating an account/wallet with Greenstand.

Reply:
I also prefer the keycloak/SSO solution